reference/supported-languages.md

```diff
@@ -0,0 +1,30 @@
+# Languages/Formats OpenRewrite supports
+
+OpenRewrite is continuously adding support for new languages and formats. As of June 2023, OpenRewrite supports:
+
+## Programming languages
+
+* Java
+* Groovy/Gradle 
+* Kotlin
+* COBOL (Closed source, commercial feature)
```

Comment:
```suggestion
* COBOL (open source, but not free for commercial use)
```

Reply:
Based off on https://github.com/openrewrite/rewrite-cobol , which uses this wording as well. If changed here then perhaps we'd best be consistent there as well.